test/e2e/webhook_test.go

```diff
@@ -82,7 +82,7 @@ var _ = Describe("bundle api validating webhook", func() {
 			Expect(err).To(WithTransform(apierrors.IsNotFound, BeTrue()))
 		})
 		It("should fail the long name bundle creation", func() {
-			Expect(err).To(MatchError(ContainSubstring(fmt.Sprintf("Invalid value: %q: metadata.name in body should be at most 40 chars long", bundle.GetName()))))
+			Expect(err).To(MatchError(ContainSubstring(fmt.Sprintf("Bundle.core.rukpak.io %q is invalid: metadata.name: Too long: may not be longer than 40", bundle.GetName()))))
```

Comment:
Does this point to test brittleness? Should we update this to only check whether the error message is complaining about character length as a follow-up?

Reply:
We could -- I'm honestly a little confounded on how this test is currently passing in CI:
* Initially, the bundle name length check was part of the core webhook, and it would return the old error (40 chars long)
* At some point, we moved the name validation to the CRD schema, and so the error returned changed to what we're seeing now (may not be longer than 40). 

I'm not sure why the test wasn't failing as part of that change. It does seem to be brittle, but as I said I'm not sure why we didn't run into issues with this earlier. I can open a follow-up.

Reply:
#435